interfaces.go

```diff
@@ -94,6 +96,26 @@ type MultiTimeSeriesTable interface {
 	TableChanger
 }
 
+type FlakeSeriesTable interface {
+	Set(v interface{}) (Op, error)
+	Update(id string, m map[string]interface{}) (Op, error)
+	Delete(id string) (Op, error)
+	Read(id string, pointer interface{}) (Op, error)
```

Comment:
The `Op` interface is not really an interface unfortunately, in that you can't just make something that satisfies the interface and expect it to work as gocassa always casts it back to a `singleOp`. A `singleOp` is very rigid - it simply represents a CQL query. So it can't really be done without rewriting gocassa.

Even if it could be done easily, I'm not completely sure it is a good idea - if we fail to extract the timestamp from the id then the database will be inconsistent with some of the operations succeeding. Currently, the only way our database can become inconsistent is if we have server issues (or a blatant coding mistake), where as this way any malformed input would result in an inconsistent database.

I did consider making the methods take a `Flake` struct and then it would be up to the dao to do the string to `Flake` conversion, this way the dao only has to handle the error once at the top.